tests/scalers/rabbitmq-queue-http-regex.test.ts

```diff
@@ -22,7 +22,7 @@ test.before(t => {
 
   sh.config.silent = true
   // create deployment
-  const httpConnectionString = `http://${username}:${password}@rabbitmq.${rabbitmqNamespace}.svc.cluster.local`
+  const httpConnectionString = `http://${username}:${password}@rabbitmq.${rabbitmqNamespace}.svc.cluster.local/${vhost}`
```

Comment:
BTW, I think that the change in the code solves the problem, but I'd like to maintain and increase the e2e test suite instead of changing one for another

Reply:
I had a different point of view of that test and believed that the behavior you just described was not intended. Sure, I can undo the change and just duplicate the test for this behavior if you think that's ok.

Reply:
yes, that'd be perfect ❤️

Reply:
IDK if undo the change or simply adding other case with current behavior, but both behaviors should be tested IMO because in fact, that test discovered a breaking change.
WDYT?